app/javascript/pages/country/widgets/widget-tree-cover-gain/widget-tree-cover-gain-component.js

```diff
@@ -1,50 +1,35 @@
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
-import numeral from 'numeral';
 
 import Loader from 'components/loader/loader';
 import WidgetHeader from 'pages/country/widgets/widget-header';
 import WidgetTreeCoverGainSettings from './widget-tree-cover-gain-settings-component';
 import './widget-tree-cover-gain-styles.scss';
 
 class WidgetTreeCoverGain extends PureComponent {
-  componentDidMount() {
-    const { setInitialData } = this.props;
-    setInitialData(this.props);
-  }
-
-  componentWillUpdate(nextProps) {
-    const { settings, updateData } = this.props;
-
-    if (JSON.stringify(settings) !== JSON.stringify(nextProps.settings)) {
-      updateData(nextProps);
-    }
-  }
-
   render() {
     const {
-      location,
       locationNames,
       isLoading,
-      totalAmount,
-      percentage,
-      locations,
+      indicators,
       settings,
-      setTreeCoverGainSettingsLocation
+      getSentence,
+      setTreeCoverGainSettingsIndicator
     } = this.props;
+
     return (
       <div className="c-widget c-widget-tree-cover-gain">
         <WidgetHeader
-          title={`TREE COVER GAIN IN ${locationNames.country &&
-            locationNames.country.label}`}
+          title={`TREE COVER GAIN IN ${locationNames.current &&
```

Comment:
I have made a helper in my current PR to handle the correct showing of country, region, or subRegion. I will make a note to update in all widgets once merged.

Reply:
https://github.com/Vizzuality/gfw/pull/3130/files#diff-d9f97bffa871e26dc9fc9d732542a0d2R80